manager-bundle/skeleton/config/config.yaml

```diff
@@ -25,6 +25,15 @@ framework:
         cookie_secure: auto
         cookie_samesite: lax
     fragments: { path: /_fragment }
+    php_errors:
+        log: true
+    uid:
+        default_uuid_version: 7
+        time_based_uuid_version: 7
+    annotations:
+        enabled: false
```

Comment:
@contao/developers Is it safe to set this?

Reply:
We don't use the `uid` stuff anywhere, do we? Why would we configure anything then?

Reply:
To get rid of the deprecation message: `Since symfony/framework-bundle 6.4: Enabling the integration of Doctrine annotations is deprecated. Set the "framework.annotations.enabled" config option to false.`

Reply:
It just forces everybody to use PHP8 attributes. If they still have annotations, they would have to configure this option themselves...

Reply:
Reverted in b2fef1c8dfe4e3fd2bf56308f136e15a93591e21.